packages/ember-htmlbars/tests/integration/attrs_lookup_test.js

```diff
@@ -1,154 +0,0 @@
-import compile from 'ember-template-compiler/system/compile';
-import ComponentLookup from 'ember-views/component_lookup';
-import Component from 'ember-views/components/component';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-import EmberView from 'ember-views/views/view';
-import run from 'ember-metal/run_loop';
-
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var owner, view;
-
-import isEnabled from 'ember-metal/features';
-if (!isEnabled('ember-glimmer')) {
-  // jscs:disable
-
-QUnit.module('component - attrs lookup', {
-  setup() {
-    owner = buildOwner();
-    owner.registerOptionsForType('component', { singleton: false });
-    owner.registerOptionsForType('view', { singleton: false });
-    owner.registerOptionsForType('template', { instantiate: false });
-    owner.register('component-lookup:main', ComponentLookup);
-  },
-
-  teardown() {
-    runDestroy(owner);
-    runDestroy(view);
-    owner = view = null;
-  }
-});
-
-QUnit.test('should be able to lookup attrs without `attrs.` - template access', function() {
-  owner.register('template:components/foo-bar', compile('{{first}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{foo-bar first="first attr"}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(view.$().text(), 'first attr');
-});
-
-QUnit.test('should be able to lookup attrs without `attrs.` - component access', function() {
```

Comment:
:heavy_check_mark: it should be able to lookup attrs without `attrs.` - component access